buildpackage_forMendix.ps1

```diff
@@ -0,0 +1,15 @@
+Param(
```

Comment:
Let's move this file into our `mx_nuget.package` folder (and maybe call it just build_nuget or something).